rlclientlib/live_model.cc

```diff
@@ -127,36 +136,73 @@ namespace reinforcement_learning
   int live_model::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response& resp, api_status* status)
   {
     INIT_CHECK();
-    return _pimpl->request_multi_slot_decision(context_json, flags, resp, status);
+    return _pimpl->request_multi_slot_decision(context_json, flags, resp, live_model::default_baseline_vector, status);
   }
 
   int live_model::request_multi_slot_decision(const char * context_json, multi_slot_response& resp, api_status* status)
   {
     return request_multi_slot_decision(context_json, action_flags::DEFAULT, resp, status);
   }
 
+  int live_model::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response& resp, const int* baseline_actions, size_t baseline_actions_size, api_status* status)
+  {
+    INIT_CHECK();
+    std::vector<int> baseline_vector = _c_array_to_vector(baseline_actions, baseline_actions_size);
+    if (event_id == nullptr && flags == NULL)
+    {
+      return _pimpl->request_multi_slot_decision(context_json, action_flags::DEFAULT, resp, baseline_vector, status);
+    }
+    if (event_id == nullptr)
+    {
+      return _pimpl->request_multi_slot_decision(context_json, flags, resp, baseline_vector, status);
+    }
+    if (flags == NULL)
+    {
+      return _pimpl->request_multi_slot_decision(event_id, context_json, action_flags::DEFAULT, resp, baseline_vector, status);
+    }
+    return _pimpl->request_multi_slot_decision(event_id, context_json, flags, resp, baseline_vector, status);
+  }
+
   int live_model::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
   {
-	  INIT_CHECK();
-	  return _pimpl->request_multi_slot_decision(event_id, context_json, flags, resp, status);
+    INIT_CHECK();
+    return _pimpl->request_multi_slot_decision(event_id, context_json, flags, resp, live_model::default_baseline_vector, status);
   }
 
   int live_model::request_multi_slot_decision(const char * event_id, const char * context_json, multi_slot_response_detailed& resp, api_status* status)
   {
-	  return request_multi_slot_decision(event_id, context_json, action_flags::DEFAULT, resp, status);
+    return request_multi_slot_decision(event_id, context_json, action_flags::DEFAULT, resp, status);
```

Comment:
whitespace

Reply:
It was a tab so I changed it into a 2 spaces to match formatting in file